tockloader/jlinkexe.py

```
@@ -459,3 +459,8 @@ def run_terminal(self):
             l = stdout_line.decode("utf-8")
             if not l.startswith("###RTT Client: *"):
                 print(l, end="")
+
+    def debug(self, binary):
+        logging.error(
+            "This functionality is not implemented for JLink"
+        )
```

Need to raise an exception here as well.